src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -127,8 +184,7 @@ export const dropdownSearchReducer = (state: IDropdownSearchState = dropdownSear
       return {
         ...state,
         id: action.payload.id,
-        options: action.payload.optionsDropdown,
-        selectedOptions: new FixedQueue<IDropdownOption>([defaultSelectedOption], 1),
+        options: action.payload.optionsDropdown ? action.payload.optionsDropdown.concat(defaultSelectedOption) : [defaultSelectedOption],
```

Comment:
The renaming is done, but for the defaultSelectedOption I think we might answer this corner case another time if it becomes a real issue because for now it does not affect the multiselect at all.